Tutorials/start/configuring-proxy.md

```diff
@@ -1,124 +1,42 @@
- 5. Change `NODEID` to an appropriate node ID value that is not already in use. Numbers between 0-255 are valid.
- 6. Change `COM` to match the COM port indicated while [connecting Faraday](connecting-hardware.md). `x` represents a number.
- 6. Save the file as `proxy.ini`
- 
-A proper configuration file will look similar to the configuration below. Notice the radio is `KB1LQD-1` on `COM71`.
- 
-![Proxy INI Example](images/Proxy-INI-Example.png "Proxy INI Example")
- 
-###Linux (Debian-Based)
- 
-> There is no need to change any other Proxy.ini settings unless you know what you're doing!
- 
- 1. Open `proxy.sample.ini` with a text editor i.e `gedit git/faradayrf/software/Proxy/proxy.sample.ini`
- 2. Update `CALLSIGN` Replace `REPLACEME` to match your callsign
- 3. Update `NODEID` to an appropriate node ID value that is not already in use. Numbers between 0-255 are valid.
- 4. Update `COM` to match the device dev path indicated while [connecting Faraday](connecting-hardware.md).
- 5. Save the file as `proxy.ini`
- 
-A proper configuration file will look similar to the configuration below. Notice the radio is `KB1LQD-1` on `COM71`. For Debian `COM71` would be `/dev/ttyUSB0` in this example.
- 
-###Mac OS X
- 
-> There is no need to change any other Proxy.ini settings unless you know what you're doing!
- 
- 1. Open `proxy.sample.ini` with a text editor `faradayrf/software/Proxy/proxy.sample.ini`
- 2. Update `CALLSIGN` Replace `REPLACEME` to match your callsign
- 3. Update `NODEID` to an appropriate node ID value that is not already in use. Numbers between 0-255 are valid.
- 4. Update `COM` to match the device dev path indicated while [connecting Faraday](connecting-hardware.md).
- 5. Save the file as `proxy.ini`
- 
-A proper configuration file will look similar to the configuration below. Notice the radio is `KB1LQD-1` on COM71. For Mac OS X `COM71` would be `/dev/cu.usbserial-40` as an example.
-
-## Connecting Proxy to Faraday
-
-Double-click on `proxy.py` to run it from a file explorer.
-Alternatively one can simply run proxy from command line:
-
-### Windows
-For example `python C:\faradayrf\software\Proxy\proxy.py`
-
-###Linux (Debian-Based)
-For example `python faradayrf/software/Proxy/proxy.py`
-
-###Mac OS X
-For example `python faradayrf/software/Proxy/proxy.py`
-
-###Proxy Running
-With Faraday connected and `proxy.py` properly configured you will see a screen similar to that shown below upon initialization.
-
-![Successful Proxy Connection](images/Proxy-Success-Connection.png "Successful Proxy Connection")
-
-> Once connected leave proxy running. It is a background application which provides a service to our core applications.
+## Running Proxy
+Once installed with `pip` or in editable mode, you can start proxy by simply running `faraday-proxy` from the command line. However it must be properly configured before it will run.
 
-Congratulations, Proxy is now running successfully!
+![Proxy INI](images/proxy.jpg "Proxy command prompt")
```

Comment:
Ahh got caught by `*.JPG` files and specifying links to images with `.jpg`. Moved images to lowercase names. ✔️